src/main/java/com/inventoryservice/controller/request/SubCategoryRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.inventoryservice.controller.request;
+
+import com.inventoryservice.model.Category;
+import com.inventoryservice.model.SubCategory;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.time.LocalDateTime;
+
+@NoArgsConstructor
+@Data
+public class SubCategoryRequest {
+    private String subCategoryName;
+    private Category category;
```

Comment:
why category is here ?

Reply:
As we send category object and fetching category_id from it.but now code is updated and used @Pathvariable categoryId use to find category.